File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loancharge/LoanChargeMvpView.java
package com.mifos.mifosxdroid.online.loancharge;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.client.Charges;

import java.util.List;

/**
 * Created by Rajan Maurya on 07/06/16.
 */
public interface LoanChargeMvpView extends MvpView {

    void showLoanChargesList(List<Charges> charges);

    void showFetchingError(String s);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loancharge/LoanChargeMvpView.kt
package com.mifos.mifosxdroid.online.loancharge

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.client.Charges

/**
 * Created by Rajan Maurya on 07/06/16.
 */
interface LoanChargeMvpView : MvpView {
    fun showLoanChargesList(charges: MutableList<Charges>)
    fun showFetchingError(s: String)
}